recipes/towhee/meta.yaml

```diff
@@ -0,0 +1,124 @@
+{% set name = "towhee" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bf44c3a7ccce4d57a38b7a34d72095f8142c1d1fd5d3d5c76651a370de034fc8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytorch >=1.4.0
+    - pyyaml >=5.3.0
+    - requests >=2.12.5
+    - tqdm >=4.59.0
+  run:
+    - python >=3.6
+    - pytorch >=1.4.0
```

Comment:
This is not listed upstream. Is it needed?

Reply:
It's needed, see https://github.com/towhee-io/towhee#getting-started